### PR TITLE
Added Mutesync

### PR DESCRIPTION
**Pull request by currentoor**
## Proposed change

Added docs for new Mutesync integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/49679
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2519
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/mutesync.markdown [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+The integration requires authentication to work.
+
+### Configuration via the frontend
+
+Menu: **Configuration** -> **Integrations**.
+
+Click on the `+` sign to add an integration and click on **mutesync**,
+and follow the configuration flow. After finishing, the Mutesync
+integration will be available.
```
Comment (klaasnicolaas) [inline source/_integrations/mutesync.markdown]:
```suggestion
{% include integrations/config_flow.md %}
```

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_integrations/mutesync.markdown [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+title: Mutesync
+description: Instructions on how to integrate Mutesync button with Home Assistant.
+ha_category:
+  - Network
+  - Binary Sensor
+ha_release: 0.79 # TODO which one?
```
Comment (balloob) [inline source/_integrations/mutesync.markdown]:
```suggestion
ha_release: 2021.5
```

File: source/_integrations/mutesync.markdown [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+title: Mutesync
+description: Instructions on how to integrate Mutesync button with Home Assistant.
+ha_category:
+  - Network
+  - Binary Sensor
```
Comment (balloob) [inline source/_integrations/mutesync.markdown]:
```suggestion
  - Presence Detection
```

File: source/_integrations/mutesync.markdown [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+ha_release: 0.79 # TODO which one?
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@currentoor'
+ha_domain: mutesync
+ha_ssdp: true
```
Comment (balloob) [inline source/_integrations/mutesync.markdown]:
```suggestion
```

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Pull request merged**